1565945881563-seed.ts

```diff
@@ -0,0 +1,9 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class seed1565945881563 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    queryRunner.query("INSERT INTO user (avatar) VALUES ('avatar')");
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<any> {}
```

Comment:
You also have to fill implementation for `down` if you evert want to revert migrations. More info here https://github.com/typeorm/typeorm/blob/master/docs/migrations.md

Reply:
Uhm, I get that and for a structural change there would be no doubt I would have implemented the reverting change, but since we're only making one insertion of  a user, is making this really necessary? Is it even possible? Technically in production use the user could have any id and removing all users based on the value that's set on the avatar field could potentially delete lots of data! The field is not unique!